src/actions/user.actions.js

```diff
@@ -152,11 +154,33 @@ const resetPasswordConfirm = (password, passwordConfirm, urlUid, urlToken) => {
   };
 };
 
+const getTargets = () => {
+  return async (dispatch) => {
+    const { data: targets } = await targetService.getTargets();
+    dispatch({
```

Comment:
In general we separate the Api calls and logic from the action, so it makes the code cleaner. In this case it would be something like that:
Create a const above for the action itself(outside getTargets):
const const getTargetsSuccess = targets =>  ({
      type: GET_TARGETS_SUCCESS,
      result: targets,
});

and then here you make  dispatch(getTargetsSuccess(targets));

Just a clean way to organize code, take it or leave.